docs/faq.rst

```diff
@@ -0,0 +1,114 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
```

Comment:
Can we remove the subsections for the time being?

Reply:
Done